Jenkinsfile

```diff
@@ -16,7 +16,10 @@ node {
         postgres = docker.image('postgres:9.4').run('-P -e POSTGRES_DB=htest')
         databaseUrl = "postgresql://postgres@${hostIp}:${containerPort(postgres, 5432)}/htest"
 
-        elasticsearch = docker.image('nickstenning/elasticsearch-icu').run('-P', "-Des.cluster.name=${currentBuild.displayName}")
+        elasticsearch_old = docker.image('nickstenning/elasticsearch-icu').run('-P', "-Des.cluster.name=${currentBuild.displayName}")
+        elasticsearchHost_old = "http://${hostIp}:${containerPort(elasticsearch_old, 9200)}"
+
+        elasticsearch = docker.image('hypothesis/elasticsearch').run('-P -e "discovery.type=single-node"')
```

Comment:
Is there a reason we are naming the ES 1.6 cluster but not the ES 6.2 cluster?

Reply:
The reason for naming the cluster is to prevent it from clustering together with other Es's running in other builds on Jenkins. With Es 6.2 the single-node option achieves the same thing